truss/protostuff/protostuff.go

```diff
@@ -0,0 +1,196 @@
+// Package protostuff provides an interface for interacting with protoc
+// requiring only paths to files on disk
+
+package protostuff
+
+import (
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/golang/protobuf/proto"
+	plugin "github.com/golang/protobuf/protoc-gen-go/plugin"
+	"github.com/pkg/errors"
+
+	assets "github.com/TuneLab/go-truss/truss/template"
+)
+
+// GeneratePBDotGo creates .pb.go files from the passed protoPaths and writes
+// them to outDir. These files import Google api's which will be created at
+// importDir.
+func GeneratePBDotGo(protoPaths []string, importDir, outDir string) error {
+	err := outputGoogleImport(importDir)
+	if err != nil {
+		return err
+	}
+
+	goImportPath, err := filepath.Rel(filepath.Join(os.Getenv("GOPATH"), "src"), importDir)
+	if err != nil {
+		return err
+	}
+
+	genGoCode := "--go_out=Mgoogle/api/annotations.proto=" +
+		goImportPath + "/third_party/googleapis/google/api," +
+		"plugins=grpc:" +
+		outDir
+
+	_, err = exec.LookPath("protoc-gen-go")
+	if err != nil {
+		return errors.Wrap(err, "protoc-gen-go not exist in $PATH")
```

Comment:
Maybe add "does" in here to clean up the robotic-sounding "not exist"? Or perhaps better and more human-targeted, "cannot find progoc-gen-go in PATH."